guides/v2.2/frontend-dev-guide/layouts/layout-overview.md

```diff
@@ -10,36 +10,34 @@ github_link: frontend-dev-guide/layouts/layout-overview.md
 functional_areas:
   - Frontend
 ---
-<h2>What's in this topic</h2>
+## What's in this topic
```

Comment:
Add blank lines before and after the heading.

Reply:
Fixed